tests/Mutator/DefinitionMutatorFactoryTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Nusje2000\ComposerMonolith\Tests\Mutator;
+
+use Nusje2000\ComposerMonolith\Composer\DefinitionMutatorFactory;
+use Nusje2000\DependencyGraph\Package\Package;
+use PHPStan\Testing\TestCase;
+use RuntimeException;
+
+final class DefinitionMutatorFactoryTest extends TestCase
+{
+    public function testCreateByPackage(): void
+    {
+        $factory = new DefinitionMutatorFactory();
+
+        $package = new Package('some/package', dirname(__DIR__, 2), false);
+        $factory->createByPackage($package);
+        $this->addToAssertionCount(1);
+
+        $this->expectException(RuntimeException::class);
```

Comment:
I'd expect some library-specific exception instead of general `RuntimeException`

Reply:
Ill add an interface and corresponding exceptions.

Reply:
I'll add an interface and the missing exceptions